lametro/templates/search/search.html

```diff
@@ -393,6 +381,14 @@ <h3 class="modal-links">
     var facetTerms = [];
     $.each(selectedFacets, function(facetName, facetValue) {
         topicFacets.includes(facetName) ? facetTerms.push(facetValue) : $.noop();
+
+        // Open the accordions for selected facets on page load
+        let accordionBtn = $("[data-bs-target=collapse_" + facetName + "]");
+        let accordionDiv = $("#collapse_" + facetName);
+
+        accordionBtn.removeClass("collapsed");
+        accordionBtn.attr("aria-expanded","true");  //TODO: this bit isn't working
```

Comment:
Thankfully the accordions were expanding correctly for me too, which is great. But I was concerned that the aria attribute wasn't updating and would lead to people with screen readers not being able to interact with it as expected.

But looking into it more, I was definitely just selecting `accordionBtn` incorrectly haha. This is all working as expected in one of the newer commits.